tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -153,113 +153,113 @@
-    <string name="pref_fast_use_tor_bridges_request_dialog">Requesting new Bridges</string>
+    <string name="dialog_new_tor_default_bridges_available">Use new default Tor bridges now?</string>
+    <string name="pref_fast_use_tor_bridges_transport_select">Please select obfuscation type:</string>
+    <string name="pref_fast_use_tor_bridges_enter_characters">Please enter the characters from the image</string>
+    <string name="pref_fast_use_tor_bridges_request_dialog">Requesting new Bridges…</string>
     <string name="pref_fast_use_tor_bridges_show_dialog">Your new bridges from bridges.torproject.org:</string>
     <string name="pref_fast_use_tor_bridges_add_dialog">Save</string>
     <string name="pref_fast_use_tor_bridges_close_dialog">Close</string>
-    <string name="pref_fast_unlock_host_wrong">Wrong host name!</string>
-    <string name="pref_fast_theme">Select Theme</string>
-    <string name="pref_fast_theme_titles_day_mode">Day Mode</string>
-    <string name="pref_fast_theme_titles_night_mode">Night Mode</string>
+    <string name="pref_fast_unlock_host_wrong">Wrong hostname!</string>
+    <string name="pref_fast_theme">Theme</string>
+    <string name="pref_fast_theme_titles_day_mode">Day</string>
+    <string name="pref_fast_theme_titles_night_mode">Night</string>
     <string name="pref_fast_theme_titles_auto_mode">Auto</string>
-    <string name="pref_fast_theme_titles_default_mode">System default</string>
+    <string name="pref_fast_theme_titles_default_mode">System</string>
     <string name="pref_fast_tor_bypass_lan">Bypass LAN addresses</string>
-    <string name="pref_fast_tor_bypass_lan_summ">Do not use Tor for LAN destinations and IANA reserved IP blocks</string>
-    <string name="pref_fast_language">Select Language</string>
-    <string name="pref_fast_block_http">Block http</string>
-    <string name="pref_fast_block_http_summ">Deny connection to http port 80</string>
-    <string name="pref_fast_categ_update">Update</string>
-    <string name="pref_fast_auto_update">Automatic Updates</string>
-    <string name="pref_fast_auto_update_summ">Check InviZible Pro and modules Updates daily</string>
-    <string name="pref_fast_chek_update">Check Update</string>
-    <string name="pref_fast_chek_update_summ">Check if new versions are available now</string>
-    <string name="pref_fast_through_tor_update">Update strictly through Tor</string>
-    <string name="pref_fast_through_tor_update_summ">Check updates only with Tor and when Tor is running</string>
-
-    <string name="pref_common_categ_run_with_root">Use Root Privileges</string>
+    <string name="pref_fast_tor_bypass_lan_summ">Do not use Tor for LAN destinations and IANA reserved IP blocks.</string>
+    <string name="pref_fast_language">Language</string>
+    <string name="pref_fast_block_http">Block HTTP</string>
+    <string name="pref_fast_block_http_summ">Deny HTTP connection over port 80.</string>
+    <string name="pref_fast_categ_update">New Versions</string>
```

Comment:
The idea is to avoid the dichotomy of "update" being to pull in a list of new upgrades, and "upgrade" to install those.
Also avoids lots of errors in translation where "update" doesn't have this duality.

Reply:
I don't see the words "update" or "upgrade" in the above lines. Please elaborate.

Reply:
Not saying I got it right, but I made an effort to avoid both, because an "update" is to pull in the list of available changes, and to "upgrade" is to install those.
"Upgrade" in turn has its own problems, as its connotation is that of moving to an exclusive better version of the app.
There are similar errors introduced in translation.

Reply:
```suggestion
    <string name="pref_fast_categ_update">New Version</string>
```